src/lib.rs

```diff
@@ -23,16 +23,29 @@ pub mod native;
 #[macro_use]
 extern crate log;
 
-/// API Interaction
-pub mod api;
-/// BlockStores
-pub mod blockstore;
-/// CAR Format Parsing
-pub mod car;
-/// FileSystem
-pub mod filesystem;
-/// Utilities
-pub mod utils;
-
+pub(crate) mod api;
+pub(crate) mod blockstore;
+pub(crate) mod car;
+pub(crate) mod filesystem;
+pub(crate) mod utils;
 #[cfg(target_arch = "wasm32")]
-pub mod wasm;
+pub(crate) mod wasm;
+
+pub mod prelude {
+    pub mod api {
+        pub use crate::api::*;
```

Comment:
Good but be sure to be explicit about whats being brought in -- no wildcard here, its ok to be verbose

Reply:
I.e by importing banyan::prelude::api::* in a downstream crate I should get Client, Account, Bucket, etc.